packages/manager/apps/pci-private-registry/src/pages/CIDR/useDatagridColumn.tsx

```diff
@@ -88,7 +89,7 @@ export const useDatagridColumn = () => {
         ) : (
           <div>{capitalizeAndJoin(props.authorization)}</div>
         ),
-      label: t('private_registry_cidr_authorization'),
+      label: ((<AuthorizationLabel />) as unknown) as string,
```

Comment:
I can't understand clearly why you use here a React Component and cast it as a string?

Reply:
Yes, agree, @ppprevost we should make the change into MRC library 👍🏻